src/reducers/app/reducers.ts

```diff
@@ -93,13 +90,16 @@ const initialState: AppState = {
     operations: [],
     accounts: [],
   },
-  availableValues: [],
+  availableValues: {
```

Comment:
ideally this should come from the metadata service

Reply:
Right, we are getting the values using `getAttributeValues` function of the  `ConseilMetadataClient`, now.

Reply:
I meant use `getEntities()`